### ISSUE TITLE
docs(contributing): create story issue template

### ISSUE THREAD

**Issue by Zaba505** [z5labs/megamind#55]
Based on discussion #54 a more agile development is being taken for managing Megamind. A part of that will be using specific issues types, called stories, to track work. Reiterating from discussion #54, the story creation process will then flow as follows:

1. User creates generic issue (ideally following the conventional commit styling)
    a. Project members who are experienced in our agile flow may skip this and instead head straight to step 2
2. Project member reviews generic issue and if its determined to add value, then a story issue is created to replace it.
3. The new story will be assigned to the [Megamind Refinement](https://github.com/orgs/z5labs/projects/7) project.
    a. Then project members will "regularly" meet to refine the stories until they believe anyone can read the story and complete the work
4. Once the story is ready it will be transitioned to the main project board, [Megamind](https://github.com/orgs/z5labs/projects/3/views/1) and assigned to one of the milestones
5. At this point, the story will just remain in the project backlog until someone decides to pick it up in a sprint
